packages-ts/starknet-gauntlet-cli/src/index.ts

```diff
@@ -57,6 +71,26 @@ const registerExecuteCommand = <UI, CI>(
   return registerCommand(deps)
 }
 
+const registerEVMExecuteCommand = <UI, CI extends Iterable<any>>(
+  registerCommand: (deps: EVMDependencies) => EVMCommandCtor<EVMExecuteCommandInstance<UI, CI>>,
+  gauntletConfig,
+) => {
+  const deps: EVMDependencies = {
+    logger: logger,
+    prompt: prompt,
+    makeEnv: (flags) => {
+      return {
+        providerUrl:
+          process.env.NODE_URL || 'https://goerli.infura.io/v3/7c43471f9d604276a856f0cff1edb645',
```

Comment:
Take a look at [this comment](https://github.com/smartcontractkit/chainlink-starknet/pull/73#discussion_r964625372) from the Bridge commands, should apply here as well

Reply:
This change is from this PR https://github.com/smartcontractkit/chainlink-starknet/pull/73 and should go away once we merge that other PR in.